settings.gradle.kts

```diff
@@ -9,6 +13,12 @@ pluginManagement {
         maven("https://repo.spongepowered.org/maven/")
         maven("https://repo.nea.moe/releases")
         maven("https://repo.sk1er.club/repository/maven-releases/")
+        maven("https://maven.deftu.xyz/releases")
+        maven("https://jitpack.io") {
+            content {
+                includeGroupByRegex("(com|io)\\.github\\..*")
```

Comment:
is this not just all of jitpack?

Reply:
It makes other artifact coordinates skip that repo

Reply:
ah okay